src/core/util/util.js

```diff
@@ -334,15 +334,16 @@ angular.module('material.core')
     },
 
     /**
-     * Give optional properties with no value a boolean true by default
+     * Give optional properties with no value a boolean true if attr provided or false otherwise
      */
     initOptionalProperties: function (scope, attr, defaults ) {
        defaults = defaults || { };
        angular.forEach(scope.$$isolateBindings, function (binding, key) {
+         var normalizedAttrName = attr.$normalize(binding.attrName);
```

Comment:
This is redundant (I don't know why it was there in the first place).
`binding.attrName` is already normalized.